Country.java

```diff
@@ -0,0 +1,49 @@
+package firstPackage;
+
+import java.util.*;
+public class Country 
+{
+	public static void main(String[] args) 
+	{
+		//Initialize ArrayList
+		List<List<String>> paths = new ArrayList<>();
+		
+		// add array values
+		paths.add(Arrays.asList("London", "New York"));
+		paths.add(Arrays.asList("New York", "Lima"));
+		paths.add(Arrays.asList("Lima", "Sao Paulo"));
+		
+		// Call FindDestination method
+		
+		String destination = findDestination(paths);
+		
+		//Print destination City
+		System.out.println(destination);
+	}
+
+	
+	public static String findDestination(List<List<String>> paths) 
+	{
+		//Declare set variable
+		Set<String> startCities = new HashSet<>();
```

Comment:
Why set and not any other like ArrayList

Reply:
Applied ArrayList